testsuite/integration-arquillian/tests/base/pom.xml

```diff
@@ -199,5 +199,12 @@
         </plugins>
 
     </build>
-    
+    <profiles>
```

Comment:
There is a separate JIRA to fix this test. We should not exclude it like this, but rather the test needs fixing.